airflow/dags/sync_kernel_to_website.py

```diff
@@ -467,9 +473,14 @@ def try_register_issues(
         if journal_id is not None:
             data = fetch_data(issue_id)
             try:
-                issue = issue_factory(
-                    data, journal_id, get_issue_order(issue_id)
-                )
+                if issue_id == get_aop_id(issue_id):
```

Comment:
@jamilatta olha a linha 471, em que é chamado o `get_journal_id(issue_id)`.
A  função `_journal_id` só considera buscar o id do journal pelos id de issues. Então, não vai retornar o journal_id para o ahead.

Reply:
@jamilatta nós não devemos cadastrar os `ahead of print` na lista de `items` dos periódicos, sendo assim a roberta está correta, essa função nunca encontrará o AOP na lista de `items` e consequentemente nunca retornará o id do aop.

Reply:
@robertatakenaka @joffilyfe 

Estou olhando para esse caso para ter um melhor entendimento!

Reply:
@robertatakenaka @joffilyfe 

Verdade, porém acredito que isso não esteja sendo contemplado no momento de entrada dos fascículos no Kernel.

Reply:
A @patymori está fazendo essa atividade (#123) @jamilatta então logo teremos o fluxo completo.

Reply:
🚀

Reply:
@joffilyfe 

Acho que durante nossas conversas não me atentei que o bundle de AOP não estaria referênciado na lista de **items** do **Jounal**. 

Somente para confirma: é mais simples ainda não é necessário navegar pelos itens, na verdade é somente obter a chave **aop** no Journal e realizar o cadastro do bundle como fascículo e os repectivos documentos como artigos.

Confirma?

Reply:
Quando eu sugeri a filtragem dos ahead of prints eu estava pensando em criar uma variável a ser passada via `xcom` com as informações obtidas [aqui](https://github.com/scieloorg/opac-airflow/blob/master/airflow/dags/sync_kernel_to_website.py#L354).

Na task de registro de issue nós filtraríamos a [lista](https://github.com/scieloorg/opac-airflow/blob/master/airflow/dags/sync_kernel_to_website.py#L513) de issues e de ahead e ai sim nós poderíamos registrar a segunda lista utilizando os mesmos métodos mas declarando que a lista é de `ahead`.

Eu acredito que esse método funciona porque:
1) Ao registrar um novo `ahead` nós temos uma modificação no `journal`;
1.1) Se torna possível detectar que aquele `bundle` é um ahead e terá um tratamento especial.
2) Quando nós tivermos uma atualização do `bundle` ele será tratado como um `bundle` qualquer e seguirá o fluxo padrão (essa parte ainda não funciona #126). 

Essa pode não ser a melhor abordagem pra seguirmos então trate apenas como uma sugestão.